app/config/services.php

```diff
@@ -18,6 +23,24 @@
  */
 
 $container = new ContainerBuilder();
```

Comment:
Van wat ik begrepen heb, werkt Dependency Injection en containers, **dus** ongeveer als volgt:

Je creert een container, waarin verschillende 'services' staan. Deze services worden vereist door bepaalde classes, maar je stopt ook deze classes in de container(?), zodat wanneer de class een bepaalde service nodig heeft, de container deze kan genereren.

Dit heb ik als voorbeeld gezien op internet:

```
<?php
class Container
{
  public function getMailTransport()
  {
    return new Zend_Mail_Transport_Smtp('smtp.gmail.com', array(
      'auth'     => 'login',
      'username' => 'foo',
      'password' => 'bar',
      'ssl'      => 'ssl',
      'port'     => 465,
    ));
  }
 
  public function getMailer()
  {
    $mailer = new Zend_Mail();
    $mailer->setDefaultTransport($this->getMailTransport());
 
    return $mailer;
  }
?>
```
Hierbij heeft de $mailer een transport nodig. Maar omdat ze nu in dezelfde container zitten, kan de transport gegenereerd worden met ```getMailTransport()```wanneer de mailer gemaakt wordt.